apps/web/src/components/FiatOnRampModal/FiatOnRampModal.tsx

```diff
@@ -279,11 +249,15 @@ export const FiatOnRampModal = memo<InjectedModalProps & FiatOnRampProps>(functi
         // @ts-ignore
         const MC_WIDGET = window?.mercuryoWidget
         MC_WIDGET.run({
-          widgetId: MERCURYO_WIDGET_ID,
+          widgetId: '3b8eed96-f637-4c90-9e4f-661fc326223b',
```

Comment:
Is this Prod widgetId?
How about `fetchLimitOfMer` are we still using this?

Reply:
If can create a new widgetId, and set in env file.

Reply:
> If can create a new widgetId, and set in env file.

this will be removed just before deploying to prod.this id is so we dont get a cors iframe error on the deployment url as

Reply:
here?